pkg/install/v1alpha1/jiva_volume_0.7.0.go

```diff
@@ -23,13 +23,7 @@ func JivaVolumeArtifactsFor070() (list ArtifactList) {
 	return
 }
 
-// jivaVolumeYamlsFor070 returns all the yamls related to jiva volume in a
-// string format
-//
-// NOTE:
-//  This is an implementation of MultiYamlFetcher
-func jivaVolumeYamlsFor070() string {
-	return `
+const jivaRunTask = `
```

Comment:
you may want to define the const with the version number. Also, a similar change to run task will be required for 0.6.0 also.. i think.

Reply:
yes I think we should move all templates to constant.